<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="verticalGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="gridGroupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>3</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QSlider" name="moveYSlider">
            <property name="minimum">
             <number>-200</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="turnXBox">
            <property name="minimum">
             <number>-360</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSlider" name="turnYSlider">
            <property name="minimum">
             <number>-360</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Turn z</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSlider" name="turnXSlider">
            <property name="minimum">
             <number>-360</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Turn y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="moveXSlider">
            <property name="minimum">
             <number>-200</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Scale</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QSpinBox" name="turnZBox">
            <property name="minimum">
             <number>-360</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="moveZBox">
            <property name="minimum">
             <double>-2.000000000000000</double>
            </property>
            <property name="maximum">
             <double>2.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Turn x</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Move y</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Move z</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QDoubleSpinBox" name="scaleBox">
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSlider" name="scaleSlider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="moveXBox">
            <property name="minimum">
             <double>-2.000000000000000</double>
            </property>
            <property name="maximum">
             <double>2.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Move x</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="moveYBox">
            <property name="minimum">
             <double>-2.000000000000000</double>
            </property>
            <property name="maximum">
             <double>2.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSlider" name="turnZSlider">
            <property name="minimum">
             <number>-360</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QSpinBox" name="turnYBox">
            <property name="minimum">
             <number>-360</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="moveZSlider">
            <property name="minimum">
             <number>-200</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="horizontalGroupBox">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_projection">
            <property name="text">
             <string>Projection</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="button_parallel">
            <property name="text">
             <string>Parallel</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="button_perspective">
            <property name="text">
             <string>Perspective</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gridGroupBox">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="label_edges_width">
            <property name="text">
             <string>Edges width</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="slider_edges_width">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>250</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_vertices_type">
            <property name="text">
             <string>Vertices type</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_vertices_width">
            <property name="text">
             <string>Vertices width</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="slider_vertices_width">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_edges_type">
            <property name="text">
             <string>Edges type</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="box_vertices_type">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Square</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Circle</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="box_edges_type">
            <item>
             <property name="text">
              <string>Solid</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dotted</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gridGroupBox1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Edges</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="vertices_color">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Black</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>White</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Red</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Orange</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Yellow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Green</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Blue</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Purple</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pink</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="edges_color">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string>Black</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>White</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Red</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Orange</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Yellow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Green</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Blue</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Purple</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pink</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Vertices</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelRed">
            <property name="text">
             <string>Background</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="background_color">
            <item>
             <property name="text">
              <string>Black</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>White</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Red</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Orange</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Yellow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Green</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Blue</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Purple</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pink</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="MyOpenGLWidget" name="picture" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuMake_screenshot">
     <property name="title">
      <string>Make screenshot</string>
     </property>
     <addaction name="actionbmp"/>
     <addaction name="actionjpeg"/>
    </widget>
    <addaction name="actionLoad_file"/>
    <addaction name="menuMake_screenshot"/>
    <addaction name="actionMake_video"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_file">
   <property name="text">
    <string>Load file</string>
   </property>
  </action>
  <action name="actionbmp">
   <property name="text">
    <string>bmp</string>
   </property>
  </action>
  <action name="actionjpeg">
   <property name="text">
    <string>jpeg</string>
   </property>
  </action>
  <action name="actionMake_video">
   <property name="text">
    <string>Make video</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">myopenglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
